test_regress/t/t_hier_block_type_param_notfound_bad.out

```diff
@@ -0,0 +1,12 @@
+%Error: Cannot open file containing hierarchical parameter declarations: '/does-not-exist'/does-not-exist
```

Comment:
Note filename is there twice, don't pass filename in the parsing call.  Also I noticed no '' were in the original message. I pushed changing this to master so please merge from master into this pull.

Reply:
Fixed here: https://github.com/verilator/verilator/pull/5333/commits/76adb8098f6bff16631c4f8c3192f8244c84a875